Packs/FeedFeodoTracker/Integrations/FeedFeodoTrackerIPBlocklist/FeedFeodoTrackerIPBlocklist.py

```diff
@@ -0,0 +1,66 @@
+from CommonServerPython import *
+
+
+def main():
+    params = {k: v for k, v in demisto.params().items() if v is not None}
+    chosen_urls = []
+    params['feed_url_to_config'] = {}
+    sources = params['feed_source']
+    if 'Last 30 Days' in sources:
+        params['feed_url_to_config']['https://feodotracker.abuse.ch/downloads/ipblocklist.csv'] = {
+            "indicator_type": FeedIndicatorType.IP,
+            "indicator": {
+                "regex": r"^.+,(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})",
+                "transform": "\\1"
+            },
+            "fields": [{
+                'creationdate': {
+                    "regex": r"^(\d{4}-\d{2}-\d{2}\s\d{2}:\d{2}:\d{2})",
+                    "transform": "\\1"
+                },
+                "port": {
+                    "regex": r"^.+,.+,(\d{1,5}),",
+                    "transform": "\\1"
+                },
+                "updatedate": {
+                    "regex": r"^.+,.+,.+,(\d{4}-\d{2}-\d{2})",
+                    "transform": "\\1"
+                },
+                "malwarefamily": {
+                    "regex": r"^.+,.+,.+,.+,(.+)",
+                    "transform": "\\1"
+                }
+            }],
+        }
+        chosen_urls.append('https://feodotracker.abuse.ch/downloads/ipblocklist.csv')
+
+    if 'Currently Active' in sources:
+        params['feed_url_to_config']["https://feodotracker.abuse.ch/downloads/ipblocklist_recommended.txt"] = {
+            "indicator_type": FeedIndicatorType.IP,
+            "indicator": {
+                "regex": r"^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})",
+                "transform": "\\1"
+            },
+            "ignore_regex": '#*'
+        }
+        chosen_urls.append('https://feodotracker.abuse.ch/downloads/ipblocklist_recommended.txt')
+
+    params['ignore_regex'] = '#'
+    params['url'] = chosen_urls
+    params['custom_fields_mapping'] = {
```

Comment:
will it cause problem if just the recommended feed is on?

Reply:
I don't think so - this is the code that creates the custom fields in the HTTPModule:
![image](https://user-images.githubusercontent.com/50295826/74231685-d09b4780-4ccf-11ea-85dd-287faf4adb5d.png)
if the mapping exists but does not find any field (such is the case here) it will simply have an empty map.